docs/developers/styleguide.rst

```diff
@@ -324,11 +324,13 @@ Quoting
         <a href="{% url 'whatever' %}" class="highlight">
 
 
-
         <!-- Bad -->
+        <a href='{% url "whatever" %}' class='highlight'>
+
+    .. code-block:: text
+
         <a href="{% url "whatever" %}" class="highlight">
         <a href='{% url 'whatever' %}' class='highlight'>
-        <a href='{% url "whatever" %}' class='highlight'>
```

Comment:
The HTML parser rejects it.  And when it rejects it the parsing fails and you don't get highlighting. Is the commit message not clear enough?

> Docs: invalid HTML cannot be highlighted
Place the invalid HTML in a plain text rendering section.